src/Community/clique_percolation.jl

```diff
@@ -42,13 +42,15 @@ CliquePercolation(;k=3) = CliquePercolation(k)
   kcliques = filter(x->length(x)>=alg.k, maximal_cliques(g))
   nc = length(kcliques)
   # graph with nodes represent k-cliques
-  h = Graph(nc)
+  h = SimpleGraph(nc)
   # vector for counting common nodes between two cliques efficiently
   x = falses(nv(g))
   for i = 1:nc
     x[kcliques[i]] .= true
     for j = i+1:nc
-        sum(x[kcliques[j]]) >= alg.k-1 && add_edge!(h, i, j)
+        if sum(x[kcliques[j]]) >= alg.k-1 
+            add_edge!(h, i, j)
+        end
     end
```

Comment:
Where I saw it, I took the opportunity to fix design issues. Here we had a mutation on the RHS of a short-circuit which is a no-no.